app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
And here, please implement your association using this approach `belongs_to :author, class_name: 'User'`